client/js/src/main/scala/zio/zmx/client/frontend/views/ChartView.scala

```diff
@@ -146,8 +155,54 @@ object ChartView {
         div(
           cls := "w-1/2 h-full p-3 ml-2",
           span(
-            cls := "text-2xl font-bold",
-            "Some Diagram info"
+            cls := "text-xl font-bold",
+            "Diagram Config"
+          ),
+          form(
+            cls := "text-2xl m-2",
+            onSubmit.preventDefault
+              .mapTo(zipVar.now()) --> (zip => {
+              println(s"HERE ${zip}")
+              // I need to generate a
```

Comment:
I need to generate a unique `DiagramConfig` in order update the diagram in the state exposed by the Command service. The questions this is from where this [UUID](https://github.com/blended-zio/zio-zmx/blob/060a8c625f2a733df692a413bc753e40643203db/client/js/src/main/scala/zio/zmx/client/frontend/model/DiagramConfig.scala?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L15) would come in handy

Reply:
No need to add, using the same one. The problem I am having here is that UUID is not exposed in this closure.